src/groundlight/client.py

```diff
@@ -95,6 +106,20 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
                     ),
                 ) from e
 
+        should_disable_tls = disable_tls_verification or os.environ.get(DISABLE_TLS_VARIABLE_NAME, None) is not None
```

Comment:
This logic isn't quite right.  To be consistent with the API_TOKEN we should check if the specified variable is `None` and if so, read the environment variable.

Environment variables are strings, so we should parse them with `bool(int(os.environ.get(DISABLE_TLS_VARIABLE_NAME, 0)))`.  If we just interpret the string as a bool, then any string will be considered `True`, even the string "no" or "False" or "0".